src/UITests/GettingStartedPage.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using Axe.Windows.Automation;
+
+namespace UITests
+{
+    [TestClass]
+    public class GettingStartedPage : AIWinSession
+    {
+        [TestMethod]
+        public void VerifyGettingStartedTitle()
+        {
+            Assert.AreEqual(session.Title, "Accessibility Insights for Windows - ");
+        }
+
+        [TestMethod]
+        public void VerifyAccessibility()
+        {
+            var result = SnapshotCommand.Execute(new Dictionary<string, string>
+            {
+                { CommandConstStrings.TargetProcessId, testAppProcessId.ToString() },
+            });
+            Assert.AreEqual(result.ScanResultsPassedCount, 0);
```

Comment:
Flipped here as well

Reply:
Parameters are correct

---
In reply to: [286113164](https://github.com/microsoft/accessibility-insights-windows/pull/362#discussion_r286113164) [](ancestors = 286113164)

Reply:
It was fixed; comment is resolved